### DISCUSSION TITLE
Correct version of distributed replication protocol

### DISCUSSION THREAD

**Discussion by timw** [orientechnologies/orientdb#9778] [Q&A] [answered]
I noticed that there is a `distributed.replicationProtocol.version` configuration option, which is documented as:

> 1 for legacy replication model (v 3.0 and previous), 2 for coordinated replication (v 3.1 and next)

The default in the code has remained at 1 however, and the only mention I can find of this feature is a brief discussion of "distributed coordination" in the 3.1 "What's new".

Is the v2 protocol considered stable, or should it be avoided?

**Comment by tglman** [comment, marked as answer]
Hi, 

At the end v2 never materialized, so just keep the current version(1) we may remove the option soon has no meaning.

Regards
